### PR TITLE
Added AUTO_INCREMENT to the docs

### PR DESCRIPTION
**Pull request by paulovitorbal**
Just got an answered that helped me on the issues and I put it here. This will help others.

### CODE REVIEW

File: doc/book/sql-ddl.md [hunk 1]
```diff
@@ -56,6 +56,13 @@ $table->addConstraint(
     new Constraint\UniqueKey(['name', 'foo'], 'my_unique_key')
 );
 ```
+You can also use the auto_increment for MySQL:
```
Comment (froschdesign) [inline doc/book/sql-ddl.md]: Please update the sentence and add backticks:

> You can also use the `AUTO_INCREMENT` attribute for MySQL:

Thanks!

**Comment by paulovitorbal** [comment]
I make the changes that you suggested and also added all the types of columns available on the `Zend\Db\Sql\Ddl\Column` namespace.

File: doc/book/sql-ddl.md [hunk 2]
```diff
@@ -146,17 +146,21 @@ In alphabetical order:
 Type             | Arguments For Construction
 -----------------|---------------------------
 BigInteger       | `$name`, `$nullable = false`, `$default = null`, `array $options = array()`
+Binary           | `$name`, `$length`, `nullable = false`, `$default = null`, `array $options = array()`
```
Comment (froschdesign) [inline doc/book/sql-ddl.md]: Please undo this changes, because it doesn't belong to the topic of this PR. Create a new PR. Thanks!

File: doc/book/sql-ddl.md [hunk 3]
```diff
@@ -146,17 +146,21 @@ In alphabetical order:
...
 Blob             | `$name`, `$length`, `nullable = false`, `$default = null`, `array $options = array()`
 Boolean          | `$name`
 Char             | `$name`, `length`
 Column (generic) | `$name = null`
 Date             | `$name`
+DateTime         | `$name`
```
Comment (froschdesign) [inline doc/book/sql-ddl.md]: The same here: undo!

File: doc/book/sql-ddl.md [hunk 4]
```diff
@@ -146,17 +146,21 @@ In alphabetical order:
...
 Decimal          | `$name`, `$precision`, `$scale = null`
 Float            | `$name`, `$digits`, `$decimal` (Note: this class is deprecated as of 2.4.0; use Floating instead)
 Floating         | `$name`, `$digits`, `$decimal`
 Integer          | `$name`, `$nullable = false`, `default = null`, `array $options = array()`
 Text             | `$name`, `$length`, `nullable = false`, `$default = null`, `array $options = array()`
 Time             | `$name`
+Timestamp        | `$name`
```
Comment (froschdesign) [inline doc/book/sql-ddl.md]: The same here: undo!

File: doc/book/sql-ddl.md [hunk 5]
```diff
@@ -56,7 +56,7 @@ $table->addConstraint(
     new Constraint\UniqueKey(['name', 'foo'], 'my_unique_key')
 );
 ```
-You can also use the auto_increment for MySQL:
+You can also use the `AUTO_INCREMENT` for MySQL:
```
Comment (froschdesign) [inline doc/book/sql-ddl.md]: Please add the word "attribute" after `AUTO_INCREMENT`.

**Comment by ezimuel** [comment]
@paulovitorbal thanks for your contribution!
